src/Modules/OrchardCore.Commerce/Migrations/ProductMigrations.cs

```diff
@@ -21,6 +23,10 @@ public int Create()
     {
         _contentDefinitionManager
             .AlterPartDefinition(nameof(ProductPart), builder => builder
```

Comment:
I couldn't do `.AlterPartDefinition<ProductPart>` here, since then I couldn't chain `.Attachable()` and `.WithDescription()`